src/display/editor/annotation_editor_layer.js

```diff
@@ -572,7 +572,8 @@ class AnnotationEditorLayer {
       return;
     }
 
-    this.#allowClick = !this.#uiManager.hasActive();
```

Comment:
Do we still need that `hasActive`-method, since it seems that there's only a single call-site left and we could just as easily inline the check instead?

Basically, we could change https://github.com/mozilla/pdf.js/blob/a9fc8792c875baeec214bd1c1e5f2e2d1303e4bc/src/display/editor/tools.js#L941 to
```javascript
this.#dispatchUpdateStates({ hasSelectedEditor: this.#activeEditor !== null });
```
and remove
https://github.com/mozilla/pdf.js/blob/a9fc8792c875baeec214bd1c1e5f2e2d1303e4bc/src/display/editor/tools.js#L961-L967